SofaKernel/modules/SofaBaseMechanics/MechanicalObject.h

```diff
@@ -96,19 +96,11 @@ class MechanicalObject : public sofa::core::behavior::MechanicalState<DataTypes>
 public:
     virtual void parse ( core::objectmodel::BaseObjectDescription* arg ) override;
 
-#ifdef SOFA_HAVE_NEW_TOPOLOGYCHANGES
+#ifdef SOFA_HAVE_NEW_TOPOLOGYCHANGES    
```

Comment:
Looks like you slipped on the space bar here :wink:

Reply:
is there an easy way to remove that without doing a commit only for that?

Reply:
Unfortunately no.